homeassistant/components/roborock/image.py

```diff
@@ -114,14 +128,15 @@ def _create_image(self, map_bytes: bytes) -> bytes:
 
 
 async def create_coordinator_maps(
-    coord: RoborockDataUpdateCoordinator,
+    coord: RoborockDataUpdateCoordinator, hass: HomeAssistant
 ) -> list[RoborockMap]:
     """Get the starting map information for all maps for this device. The following steps must be done synchronously.
 
     Only one map can be loaded at a time per device.
     """
     entities = []
-
+    assert coord.config_entry is not None
```

Comment:
I know I have been asked recently not to use assert. But I need to look more into this. config_entry can be None, but i'm not sure if that is a typing thing or a real possibility

Reply:
Coordinators created as part of `async_setup_entry` will always have a config entry set. You can change the typing on your coordinator using `config_entry: ConfigEntry`